### DISCUSSION TITLE
middleware/elasticsearch/user-guide/basic/

### DISCUSSION THREAD

**Discussion by giscus** [DaoCloud/DaoCloud-docs#1870] [Announcements]
# middleware/elasticsearch/user-guide/basic/

DaoCloud Enterprise 5.0 下一代容器化平台集大成者，引领云原生浪潮，推动数字化转型。

https://docs.daocloud.io/middleware/elasticsearch/user-guide/basic/

**Comment by liyuerich** [comment]
这个概览的图已经旧了，能否更新？

**Reply by Michelle951** [reply]
fixed by Pull Request: https://github.com/DaoCloud/DaoCloud-docs/pull/1881
